nengo_fpga/networks/fpga_pes_ensemble_network.py

```diff
@@ -177,25 +200,15 @@ def connect_thread_func(self):
         ssh_channel = self.ssh_client.invoke_shell()
 
         # If board configuration specifies using sudo to run scripts
-        if fpga_config.getboolean(self.fpga_name, 'use_sudo') and \
-           fpga_config.get(self.fpga_name, 'user') != 'root':
+        # - Assume all non-root users will require sudo to run the scripts
+        # - Note: Also assumes that the fpga has been configured to allow
+        #         the ssh user to run sudo commands WITHOUT needing a password
+        #         (see specific fpga hardware docs for details)
```

Comment:
Add this modification to the SD card image. There are instructions on the development page of the docs (currently commented out but the source is there)

Reply:
The only image that needs to be modified is the pynq image. The pynq docs will be updated to reflect this new assumption.